src/zzz_od/application/charge_plan/charge_plan_app.py

```
@@ -194,16 +183,20 @@ def notorious_hunt(self) -> OperationRoundResult:
     @node_from(from_name='恶名狩猎', success=False)
     @operation_node(name='挑战完成')
     def challenge_complete(self) -> OperationRoundResult:
-        # 挑战成功后，重置last_tried_plan以继续查找下一个任务
+        # 成功后继续正常轮转；失败则标记当前计划已跳过，避免在同一轮里死循环重试
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)